sui/rotate.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+
+# Environment Variables
+
+NAMESPACE="testnet"
+NAMESPACE2="testnet-amplifiers"
+WORKER="ampd-axelar-amplifier-worker-0"
+BASTION="axelar-core-bastion-bastion-6f5ddc97c5-jl45c"
+BASTION_CONTAINER="axelar-core-bastion"
+SERVICE_NAME="amplifier"
+MULTISIG_PROVER_ADDRESS="axelar1v8jrupu2rqpskwgtr69max0ajul92q8z5mdxd505m2hu3xc5jzcqm8zyc6"
+VOTING_VERIFIER_ADDRESS="axelar1sykyha8kzf35kc5hplqk76kdufntjn6w45ntwlevwxp74dqr3rvsq7fazh"
+SERVICE_REGISTRY="axelar1rpj2jjrv3vpugx9ake9kgk3s2kgwt0y60wtkmcgfml5m3et0mrls6nct9m"
+
+# On mainnet there will be a rotation delay, so we can only run one iteration at a time
+ITERATIONS=1
+
+print_section() {
+    echo ""
+    echo ""
+    echo "============================== $1 =============================="
+}
+
+print_success() {
+    echo ""
+    echo "[✔] $1"
+}
+
+print_info() {
+    echo ""
+    echo "[INFO]: $1"
+}
+
+# Functions
+set_namespace() {
+    print_section "Setting Namespace"
+    kubectl config set-context --current --namespace="$NAMESPACE"
+    print_success "Namespace set to $NAMESPACE"
+}
+
+query_current_verifier_set() {
+    kubectl exec -it "$BASTION" -c "$BASTION_CONTAINER" -- bash -c "axelard q wasm contract-state smart $1 '\"current_verifier_set\"'"
+}
+
+query_next_verifier_set() {
```

Comment:
Avoid creating bash scripts, they're not very readable and maintainable. Create a JS script instead. We already have JS examples on how to interact with Axelar contracts. This script will be useful for performing rotations for any connections, so convert to JS and make it more generic. Depending on `chainType`, it can call the `submitProof` helper from the relevant Sui/Stellar/EVM for simplicity

Reply:
The script should only do a single rotation, the caller can loop over it if needed